Dockerfile

```diff
@@ -8,23 +8,21 @@ WORKDIR /app
 
 # Run updates and cleanup
 RUN apt-get update && \
-    apt-get install -y \
-      netcat \
-      git && \
-    rm -rf /var/lib/apt/lists/*
+  apt-get install -y \
+  netcat \
+  git && \
+  rm -rf /var/lib/apt/lists/*
 
 #  Changing git URL because network is blocking git protocol...
 RUN git config --global url."https://".insteadOf git://
 RUN git config --global url."https://github.com/".insteadOf git@github.com:
 
 COPY package.json package-lock.json /app/
 
-RUN npm config set registry https://repo.datapunt.amsterdam.nl/repository/npm-group/ && \
-    npm --production=false \
-        --unsafe-perm \
-        --verbose \
-        install && \
-    npm cache clean --force
```

Comment:
You should keep this line. It keeps the image size small.

Reply:
Not sure it is necessary, this image is just the build image, the build results are copyed in the other image, this image is thrown away after each CI build, si not saved in the repository. This cleanup was handy in older versions of the dockerfile when multi images were not possible.